includes/class-recaptcha.php

```diff
@@ -0,0 +1,292 @@
+<?php
+/**
+ * Class for reCAPTCHA integration.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class for reCAPTCHA integration.
+ */
+final class Recaptcha {
+	const RECAPTCHA_SCRIPT_HANDLE = 'newspack-recaptcha';
+	const RECAPTCHA_THRESHOLD     = 0.5;
+	const RECAPTCHA_API_NAMESPACE = 'newspack/v1';
```

Comment:
We already have a global constant for the `newspack/v1` endpoint called `NEWSPACK_API_NAMESPACE`

Reply:
Ah, good call: 2bf452c